src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Pbkdf2Implementation.Android.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal partial class Pbkdf2Implementation
+    {
+        public static unsafe void Fill(
```

Comment:
The managed implementation imposes a minimum salt length that the one-shots (intentionally) do not. We could:

1. Make another non-validating constructor, or take another `bool` since the constructor is `internal`.
2. Not worry about it and let this be a quirk of Android